implementations/bovine.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "bovine",
+  "implementation": "bovine",
+  "issuers": [
+    {
+      "id": "urn:uuid:bovine:issuer:eddsa-rdfc-2022",
+      "endpoint": "https://bovine.social/w3c-di/issue/eddsa-rdfc-2022",
+      "tags": ["eddsa-rdfc-2022"]
+    },
+    {
+      "id": "urn:uuid:bovine:issuer:eddsa-jcs-2022",
+      "endpoint": "https://bovine.social/w3c-di/issue/eddsa-jcs-2022",
+      "tags": ["eddsa-jcs-2022"]
+    },
+    {
+      "id": "urn:uuid:bovine:issuer:ecdsa-rdfc-2019",
+      "endpoint": "https://bovine.social/w3c-di/issue/ecdsa-rdfc-2019",
+      "tags": ["ecdsa-rdfc-2019"]
```

Comment:
Along with the tags, ecdsa test suites also require you to specify the `supportedEcdsaKeyTypes` property. Currently the test suite supports ECDSA key types `P-256` and `P-384`.

You can specify the key types supported by your implementation in the issuer and verifier configs like this:

Example:
```
  "issuers": [{
   ...
    "supportedEcdsaKeyTypes": ["P-256"]
    "tags": ["ecdsa-rdfc-2019", "localhost"]
  }],
 "verifiers": [{
    ...
    "supportedEcdsaKeyTypes": ["P-256"]
    "tags": ["ecdsa-rdfc-2019", "localhost"]
  }]
```